asyncthreadpool.nimble

```diff
@@ -9,3 +9,4 @@ license       = "MIT"
 # Dependencies
 
 requires "nim >= 1.4.2"
+requires "chronos"
```

Comment:
I do not want to add this explicit dependency. It's fine to install chronos in the CI for testing, but not by default.

Reply:
I dont like either, as nimble dont support install flags , no solution for this.

Reply:
This seems to work:
```nim
if getEnv("ASYNCTHREADPOOLS_WITH_CHRONOS") == "YES":
  requires "chronos"
```
Then set `ASYNCTHREADPOOLS_WITH_CHRONOS=YES` in CI.

Reply:
I tried nimble did not aware this requires, so it did not add this to `--path` when run `nimble test` it fails.

Reply:
oh, I get it , it will work for user installation.